dialog/lib/dialog.ts

```diff
@@ -389,7 +393,8 @@ export class Dialog extends LitElement {
     let promise = this.updateComplete;
     if (duration > 0) {
       promise = new Promise((r) => {
-        setTimeout(r, duration);
+        // setTimeout(r, duration);
+        this.containerElement.ontransitionend = () => r(null);
```

Comment:
Can we actually revert this change and just use the promise awaiting for now?

Using `on<event>` properties is generally frowned upon, and I'd like to avoid any unknown side effects from changing this since it doesn't impact animation.